index.html

```diff
@@ -1703,11 +1703,54 @@
 
       <section><h3>Obtaining push permission</h3>
       <p>
-        To <dfn>obtain push permission</dfn>, run these steps:
+        To <dfn>obtain push permission</dfn> given <var>target</var> of type
+        <code><a>NFCPushTarget</a></code>, run these steps:
         <ol>
           <li>
             If a <a>prearranged trust relationship</a> exists,
-            then return <code>true</code>.
+            return <code>true</code>.
+          </li>
+          <li>
+            If <var>target</var> has the value <code>"peer"</code>, i.e. the
+            push is requested on an <a>NFC peer</a>, return <code>true</code>.
+            <p class="note">
+              The need for explicit user gesture for tapping an <a>NFC peer</a>,
+              together with the other <a href="#security-policies">
+              security policies</a> (the method being called from a visible
+              <a>secure context</a> which is also in focus), are deemed
+              sufficient for expressing willingness and awareness of the user
+              to do the push operation.
+            </p>
+          </li>
+          <li>
+            If <var>target</var> has the value <code>"tag"</code>, i.e. the
+            push is requested on an <a>NFC tag</a>, which contains a
```

Comment:
"i.e." means the two sides are equivalent, but 'has the value "tag"' does not imply that the tag contains a Web NFC message.